### PR TITLE
[UI] Memoized Charts And Graph Components

### LINKED ISSUE #8211: [dashboard] Unresponsive MeshModel Dashboard
Currently, when a user hovers over MeshModel charts then the page becomes unresponsive and leads to flickering of chart.

This behavior is unexpected and should be resolved for better UX for Meshery users.

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**
The Dashboard component is rerendering multiple times , so the charts as children were also getting rerendered a lot .
As chart component itself doesnt do any io so it is good candidate for memoizing .

Now the Chart Components won't be rerendered even if the Dashboard does .

Related To #8211  

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@vaibhav135 look good?

File: ui/components/DashboardComponent.js [hunk 1]
```diff
@@ -1229,6 +1229,7 @@ class DashboardComponent extends React.Component {
   };
 
   render() {
+    console.info("Rerendering Dashboard")
```
Comment (vaibhav135) [inline ui/components/DashboardComponent.js:1232]: @aabidsofi19 please remove this console if not needed.

**Review by vaibhav135 — CHANGES_REQUESTED** [review]
Just a small comment. Everything else LGTM.

File: ui/components/DashboardComponent.js (hunk 1 above)
Comment (aabidsofi19) [reply ui/components/DashboardComponent.js:1232]: I thought to keep it for an easy check to see if the dashboard rerenders are really increasing in production  .

**Review by vaibhav135 — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]

**Pull request merged**